README.md

```diff
@@ -97,116 +97,114 @@ The `plugin:regexp/all` config enables all rules. It's meant for testing, not fo
 
 ## :white_check_mark: Rules
 
-<!--RULES_SECTION_START-->
+<!-- begin auto-generated rules list -->
 
-The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) automatically fixes problems reported by rules which have a wrench :wrench: below.
-The rules with the following star :star: are included in the `plugin:regexp/recommended` config.
-
-<!--RULES_TABLE_START-->
+💼 Configurations enabled in.\
+⚠️ Configurations set to warn in.\
+✅ Set in the `plugin:regexp/recommended` configuration.\
+🔧 Automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--fix).\
+💡 Manually fixable by [editor suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).\
+❌ Deprecated.
```

Comment:
This is not a deal-breaker, but would it be possible to remove the "Deprecated" column? We have all deprecated nodes in their own category, so the column is kinda unnecessary.

Or maybe we should keep it to be more consistent with other projects using ESLint Doc Generator?

Reply:
I was thinking the same thing actually so I went ahead and removed the "Deprecated" column using the `ruleListColumns` option.

I also have an issue to consider making this automatic: https://github.com/bmish/eslint-doc-generator/issues/323

Reply:
Ohh, thank you!